### DISCUSSION TITLE
Balloon Focus capability

### DISCUSSION THREAD

**Discussion by MahdisYavari** [skydoves/Balloon#285] [Q&A]
Hello @skydoves 

I have some problem with this method
 `setFocusable(false)`
when it's true, the balloon is showing, can not touch other views
I set it to false because I don't want this happens but actually nothing works correctly . when I touch other views, first balloon dismisses and then other views can be touched. How can I solve this problem ?

**Comment by skydoves** [comment]
Hey @MahdisYavari,
Sorry for the late response. This seems related to https://github.com/skydoves/Balloon/issues/193
